assignment/README.md

```diff
@@ -1,65 +1,122 @@
+works to identify the most congested spots in the Philadelphia region, possible causes to their congestion and potential remedies.
+
+- What do they gain from your application' use?
+
+My application would help urban planners like my boss understand what could potentially be affecting the reasons why congested intersections in Philadelphia are popping up where they are.
+This would help people like him devise potential solutions to the congestion and where to best place them.
+
+- Are there any website/application examples in the wild to which you can compare your final?
+
+I see it as a expansion in JavaScript of one of the DVRPC's web maps showing where the most congested road segments in the Philadelphia are: https://dvrpcgis.maps.arcgis.com/apps/MapSeries/index.html?appid=732efbf95f76489598277df671b5d6b2
+
+#### Layouts and visual design
+
+So far, we've built all our applications with a side bar for
+representing non-map content and navigation. This is not the only
+successful design. Extra content could be displayed in a top bar,
+through side bars on both sides, and any combination of
+these as well as a number not mentioned. Try to describe your
+application's visual layout. Conceptually (no need for extensive CSS
+here), what will this design require?
+
+I am considering choosing one of two different methods: 
+
+1) Formatting the application like the mid-term where the user clicks through many slides, each detailing a different aspect of the
+data. For example, one or two slides would show the spatial relationships between the congested intersections and traffic signals, the next would show the relationships between the
+congested intersections and ramps, etc. 
+
+2) There would be different tabs the user would click on in the top bar, with each tab being like a slide in the first option, except no next or previous buttons, etc. Each tab would also
+have a sidebar that would display narrative text, interactive graphs, inputs and buttons to filter, etc. like with the first option.
+
+Also, in all maps the application will show, the congested intersections would have pop-ups detailing information such as their volume delay amount, ranking, and intersection name.
+The application would let users click on or hover over the data in the map to view information about it for example.
+
+## Anticipated difficulties
+
+Thinking about weaknesses can be useful. What do you anticipate being
+most difficult about this project? How will you attempt to cope with
+these difficulties? For example, asynchronous behavior (ajax, events)
+are hard to use and think about. Global variables are a strategy for
+coping with that difficulty by breaking data out of the asynchronous
+context.
+
+I anticipate having trouble with making tabs. Up until now, I have never made a web application with tabs in JavaScript, the most complex organization method I have used so far was putting

```

Comment:
If this does prove to be a pain, you can always have your 'tabs' simply link to other `html` documents and treat each `tab` as a separate application